code/dynamic_programming/src/rod_cutting/rod_cutting.go

```diff
@@ -0,0 +1,33 @@
+// Part of Cosmos by OpenGenus Foundation
+// Code contributed by Marc Tibbs
+
+// A Dynamic Programming solution for Rod Cutting problem 
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+const min float64 = -1 << 63
+
+func rodCut(price []float64, n int) float64 {
+	val := []float64{0}
+	val = append(val, price...)
+
+	for i := 1; i < n+1; i++ {
+		var max_val = min
+		for j := 0; j < i; j++ {
+			max_val = math.Max(max_val, price[j] + val[i-j-1])
+		}
+		val[i] = max_val
+	}
+	return val[n]
+}
+
+# Driver program to test rodCut function
```

Comment:
Wrong comment style...

Reply:
Sorry about that. I corrected the comment style